### PR TITLE
feat: aside controls variant added to carousel component

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Aside controls variant added to carousel component

Screenshot 2024-07-08 at 10 30 07 a m

### CODE REVIEW

File: @kiva/kv-components/vue/KvCarousel.vue [hunk 1]
```diff
@@ -2,11 +2,13 @@
 	<section
 		ref="rootEl"
 		class="kv-carousel tw-overflow-hidden tw-w-full"
+		:class="{ 'lg:tw-relative': asideControls }"
 		aria-label="carousel"
 	>
 		<!-- Carousel Content -->
 		<div
 			class="tw-flex tw-gap-x-4"
+			:class="{ 'tw-mx-auto lg:tw-w-[82%]': asideControls }"
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvCarousel.vue:11]: We've been trying not to use the JIT TW classes. An alternative here is to use a custom class and CSS.

File: @kiva/kv-components/vue/KvCarousel.vue [hunk 2]
```diff
@@ -72,6 +75,50 @@
...
+			>
+				<kv-material-icon
+					class="tw-w-4"
+					:icon="mdiArrowLeft"
+				/>
+				<span class="tw-sr-only">Show previous slide</span>
+			</button>
+		</div>
+		<div
+			v-if="asideControls"
```
[the remark this answers is not in the document]
Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvCarousel.vue:101]: Do you mean wrapping both of them in a single one? We'll not be able to click over cards as having an overlay element on them.

[the remark this answers is not in the document]
Comment (dyersituations) [reply @kiva/kv-components/vue/KvCarousel.vue:101]: I meant a single `<template` with the `v-if`

[the remark this answers is not in the document]
Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvCarousel.vue:101]: Oh yeah, that makes sense, thanks.

**Review by dyersituations — APPROVED** [review]

**Review by mcstover — COMMENTED** [review]
What do these look like when the page design aligns the cards at the left and right along with the primary...does that mean the first and last card are obscured by the controls?

**Comment by roger-in-kiva** [comment]
> What do these look like when the page design aligns the cards at the left and right along with the primary...does that mean the first and last card are obscured by the controls?

No, for that reason we are changing the size of content to 82% width here: https://github.com/kiva/kv-ui-elements/blob/96252d50cf564ca8a44a53dd055edc1dcb2701e4/%40kiva/kv-components/vue/KvCarousel.vue#L387

Screenshot 2024-07-08 at 12 02 39 p m

**Review by mcstover — COMMENTED** [review]
👍

**Pull request merged**